demo/mkdocs.yml

```diff
@@ -41,12 +42,16 @@ markdown_extensions:
 nav:
   - Home: index.md
   - Showcase Demonstration:
+    - Why use Seqera Platform?: intro.md
     - Getting started: demo_overview.md
     - Add a Pipeline: add_a_pipeline.md
     - Add a Dataset: add_a_dataset.md
     - Launch Pipeline: launch_pipeline.md
     - Monitor a Run: monitor_run.md
+    - Monitoring views: monitoring_views.md
     - View Run Details: run_details.md
     - Resume a Pipeline: resume_pipeline.md
     - Data Explorer: data_explorer.md
-    - Optimize your Pipeline: pipeline_optimization.md
\ No newline at end of file
+    - Data Studios: data_studios.md
+    - Optimize your Pipeline: pipeline_optimization.md
+    - Automation on Seqera Platform: automation.md
```

Comment:
Maybe add a Conclusion section to circle back to the "story" and highlight how Seqera Platform addresses bioinformaticians needs - reproducibility, collaboration, monitoring, reporting, etc.

Reply:
Great idea, added in a summary at the end.